### ISSUE TITLE
Regression: Ads not blocked on site after enabling flash

### ISSUE THREAD

**Issue by srirambv** [brave/browser-laptop#4225]
**Did you search for similar issues before submitting this one?**
Yes

**Describe the issue you encountered:**
Ads not blocked on site after enabling flash

**Expected behavior:** 
Ad should be blocked

- Platform (Win7, 8, 10? macOS? Linux distro?):
Windows 10 x32

- Brave Version:
0.12.2 RC2

- Steps to reproduce:
    1. Visit http://www.y8.com/games/superfighters
    2. Right click to enable flash
    3. Ad not blocked before loading flash conent

- Screenshot if needed:
![image](https://cloud.githubusercontent.com/assets/17010094/18781293/061ead1e-8179-11e6-9271-00a99be6f9e6.png)

- Any related issues:
#4205

**Comment by diracdeltas** [comment]
i'm inclined to mark this wontfix. y8 recently changed their ads system so that the game won't load until the ad is played so i added a sitehack to load their ads on purpose.

**Comment by bbondy** [comment]
remvoing from 0.12.4 since it's wontfix
